xblock/test/django/test_request.py

```diff
@@ -80,12 +80,12 @@ def test_headers(self):
     def test_content_type(self):
         # Default charset
         self.assertEquals(
-            self._as_django(content_type='application/json')['Content-Type'],
+            self._as_django(content_type='application/json; charset=UTF-8')['Content-Type'],
```

Comment:
Honestly, I'm having a hard time deciding on the right answer here. BUT: webob seems to have decided that character sets are wrong in a json content type.  The important thing is that these tests call this function in a way similar to how it is called in edx-platform.  I don't know what that way is, but this fix is both changing the way the function is called in the test, and avoiding the change that webob thought was important.  This worries me.